stages/stage_2/compile.py

```diff
@@ -3,14 +3,12 @@
 from clvm_tools.NodePath import LEFT, TOP
 
 from .defaults import default_macro_lookup
-from .helpers import brun, eval
+from .helpers import brun, eval, quote
 from .mod import compile_mod
 
-
 CONS_KW = KEYWORD_TO_ATOM["c"]
 QUOTE_KW = KEYWORD_TO_ATOM["q"]
```

Comment:
Any chance we can remove this line? That would give more confidence that other `QUOTE_KW` references aren't sneaking in. Probably in the other files too.